src/stories/1-intro/form-builder/components/form-builder-tool.styles.ts

```diff
@@ -111,7 +111,7 @@ export const SchemaEditor = styled(Textarea)`
 	width: 100%;
 	overflow: auto;
 	padding: 1rem;
-	border: 1px solid ${V2_Color.Neutral[5]};
+	border: 1px solid ${Colour.border};
 	border-radius: 4px;
```

Comment:
use the [Border](https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-border-introduction--docs) and [Radius](https://designsystem.life.gov.sg/react/index.html?path=/docs/foundations-radius-introduction--docs) tokens where you can